datasource/common/src/main/java/org/opennms/oce/datasource/common/InventoryObjectBean.java

```diff
@@ -98,6 +113,13 @@ public void setPeers(List<InventoryObjectPeerRefBean> peers) {
         this.peers = peers;
     }
 
+    public void setPeer(InventoryObjectPeerRefBean pRef) {
```

Comment:
I'd call this `addPeer` rather than `setPeer` since `set*` normally implies that we're overwriting the current value.